test/inductor/test_templated_attention.py

```diff
@@ -191,6 +194,52 @@ def score_mod(score, b, h, m, n):
 
         self.run_test(score_mod)
 
+    @supported_platform
+    def test_logsumexp_correctness(self):
```

Comment:
I would also test versions that only returns one of the the output buffers

Reply:
added tests that return both out, lse | just LSE and our existing tests cover the just out case